File: RelacionamentoClientes/src/br/com/inatel/pos/dm102/runner/Runner.java
package br.com.inatel.pos.dm102.runner;

import br.com.inatel.pos.dm102.bo.AtendimentoBO;

public class Runner {

	public static void main(String[] args) {
		AtendimentoBO atendimentoBO = new AtendimentoBO();
			atendimentoBO.iniciarAtendimento();
	}
}

File: RelacionamentoClientes-C#/RelacionamentoClientes/RelacionamentoClientes/Runner.cs
using RelacionamentoClientes.Business;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RelacionamentoClientes
{
    class Runner
    {
        static void Main(string[] args)
        {
            AtendimentoBO bo = new AtendimentoBO();
            bo.IniciarAtendimento();
        }
    }
}
